cmd/state-installer/installer.go

```diff
@@ -111,11 +117,9 @@ func install(installPath string, cfg *config.Instance, out output.Outputer) erro
 	}
 
 	tmpDir := filepath.Dir(exe)
-	// clean-up temp directory when we are done.
-	defer os.RemoveAll(tmpDir)
-
 	inst := installer.New(filepath.Join(tmpDir, "bin"), installPath)
 	defer func() {
+		os.RemoveAll(tmpDir)
```

Comment:
Did you remove this for debugging purposes?

Reply:
I didn't? It's just in a single defer now.

Reply:
Oh dear! How silly of me...